src/Player.js

```diff
@@ -0,0 +1,15 @@
+const Game = require('../src/Game')
+class Player {
+  constructor(name) {
+    this.name = name,
+    this.turn = false,
+    this.score = 0
+  }
+
+  // startGame(playersArr) {
```

Comment:
I see where you are going with this.... I bet it's just a small minor issue that we need to solve.